site/src/pages/DeploymentSettingsPage/NotificationsPage/Troubleshooting.tsx

```diff
@@ -29,17 +30,17 @@ export const Troubleshooting: FC = () => {
 			</div>
 			<div>
 				<span>
-					<LoadingButton
-						variant="outlined"
-						loading={isLoading}
-						size="small"
-						css={{ minWidth: "auto", aspectRatio: "1" }}
+					<Button
+						variant="outline"
+						size="sm"
```

Comment:
nit: button font is smaller according to Chromatic, is it expected?

Reply:
Let me check 👀

Reply:
Yes, it is!